.github/workflows/continuous-integration.yml

```diff
@@ -0,0 +1,15 @@
+name: Continuous Integration
+on: [pull_request]
+
+jobs:
+  build:
+    name: Run unit tests with Jest
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup GIT for testing
+        run: git config --global user.email "cplace-npm-tools@cplace.io" && git config --global user.name "cplace-npm-tools GitHub Action"
```

Comment:
not sure what this is about, do you need to set a user for unit tests?

Reply:
Yes :D I found that out - one of the test is doing a dummy init of a repo to test stuff, so it's not a plain unit test but yeah...